leakcanary-analyzer/src/main/java/com/squareup/leakcanary/AnalysisResult.java

```diff
@@ -20,16 +20,17 @@
 public final class AnalysisResult implements Serializable {
 
   public static AnalysisResult noLeak(long analysisDurationMs) {
-    return new AnalysisResult(false, false, null, null, null, analysisDurationMs);
+    return new AnalysisResult(false, false, null, null, null, 0, analysisDurationMs);
```

Comment:
I would argue that there is too many literals. Extracting local vars seems to be overkill for this though.
Maybe inline comments will make a deal? Like:

```java
return new AnalysisResult(
    false /* leakFound */,
    false /* excludedLeak */,
    null  /* className */, 
    null  /* leakTrace */, 
    null  /* failure */, 
    0     /* retainedHeapSize */, 
    analysisDurationMs);
```